truss/cli.py

```diff
@@ -210,6 +210,23 @@ def kill_all():
     truss.kill_all()
 
 
+@cli_group.command()
+@error_handling
+def cleanup() -> None:
+    """
```

Comment:
Would be good to add more details here. These comments show up as help message on the truss cli. Something like:
```
Clean up truss data.

Truss creates temporary directories for various operations such as for building docker images. This command clears that data to free up disk space.
```